open_academy/views/open_academy_views.xml

```diff
@@ -1,4 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
 <odoo>
+    <record id="open_academy_dashboard_view" model="ir.ui.view">
+        <field name="name">Dashboard</field>
+        <field name="model">board.board</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <form>
+                <board style="1">
+                    <column>
+                        <action name="%(open_academy.session_action_graph)d" string="Session Graph View"/>
```

Comment:
did you create your .po file before these changes? I didn't see them inside the file

Reply:
yes, I created  it before, in the course this chapter comes after the translations chapter

Reply:
create a new one, do not delete this only add the new terms of the new file into this one 

otherwise some terms will be on english

Reply:
I mean the .po file